.travis.yml

```diff
@@ -1,22 +1,20 @@
 language: node_js
+cache: false
 sudo: required
 script:
   - npm test
 branches:
   only:
     - master
-cache:
-  directories:
-    - node_modules
-# before_install:
-#   - docker pull selenium/standalone-chrome:
-#   - docker run -v /dev/shm:/dev/shm -e TZ=America/Chicago -d -p 4444:4444 selenium/standalone-chrome
-# before_deploy:
-#   - lerna run build:production
-# deploy:
-#   provider: pages
-#   skip_cleanup: true
-#   github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
-#   local_dir: build
-#   on:
-#     branch: master
+before_install:
+  - docker pull selenium/standalone-chrome:3.8.1-erbium
+  - docker run -v /dev/shm:/dev/shm -e TZ=America/Chicago -d -p 4444:4444 selenium/standalone-chrome:3.8.1-erbium
```

Comment:
Do we have wdio tests in terra-ui.com? I wonder, do we need them? If we don't, we could remove this docker stuff here.

Reply:
I dont think we need them in this repo.